### PR TITLE
recipes to build vis with static lpeg

### PR DESCRIPTION
**Pull request by xcko**
vis editor from https://github.com/martanne/vis
a vi like editor with sam's regular expressions.
These recipes are for necessary dependencies.
Build logs:
[build_libtermkey.log](https://github.com/sabotage-linux/sabotage/files/2236736/build_libtermkey.log)
[build_lpeg.log](https://github.com/sabotage-linux/sabotage/files/2236738/build_lpeg.log)
[build_vis.log](https://github.com/sabotage-linux/sabotage/files/2236739/build_vis.log)

### CODE REVIEW

**Comment by xcko** [comment]
updated vis log: 
[build_vis.log](https://github.com/sabotage-linux/sabotage/files/2237127/build_vis.log)

**Comment by xcko** [comment]
last update. may as well compile vis statically too since it is supported upstream.
[build_vis.log](https://github.com/sabotage-linux/sabotage/files/2237161/build_vis.log)

File: pkg/libtermkey [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+pkgver=1
+
+[deps]
+ncurses
+
+[build]
+sed -i 's/^PREFIX=.*$/PREFIX='"$butch_prefix"'/' Makefile
+
+make \
+	CFLAGS="$optcflags" \
```
Comment (rofl0r) [inline pkg/libtermkey]: i guess `make PREFIX=$butch_prefix` would've worked too (instead of the sed)

File: pkg/lpeg [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+pkgver=1
+
+[deps]
+lua53
+
+[build]
+patch -p1 < "$K"/lpeg-build-static-lib.patch
+sed -i 's/^CFLAGS =/CFLAGS +=/' makefile
+sed -i 's/^COPT =/# COPT =/' makefile
+make LUADIR=/include/lua5.3
```
Comment (rofl0r) [inline pkg/lpeg]: $butch_prefix should be used here too

**Comment by rofl0r** [comment]
thanks, merged as 
https://github.com/sabotage-linux/sabotage/commit/7c20a8dc74fd439b92c4347739c43affe3b4dc74

**Pull request closed without merging**